mlrun/api/utils/auth/verifier.py

```diff
@@ -125,6 +125,21 @@ def query_global_resource_permissions(
             raise_on_forbidden=raise_on_forbidden,
         )
 
+    def query_resource_permissions(
+        self,
+        resource_type: mlrun.api.schemas.AuthorizationResourceTypes,
+        resource_name: str,
+        action: mlrun.api.schemas.AuthorizationAction,
+        auth_info: mlrun.api.schemas.AuthInfo,
+        raise_on_forbidden: bool = True,
+    ) -> bool:
```

Comment:
So, now we have `query_global_resource_permissions` (used in marketplace APIs) and this new method - the only difference is that this one has the `name` parameter. I don't really see a reason to have both methods - this method can replace the `query_global_resource_permissions` and be called with `name=""`. Maybe change `query_global_resource_permissions` to call this one with `name=""`. WDYT?